src/state/sagas/datasets/QueryDatasetTwingraph/QueryDatasetTwingraph.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { all, call, delay, put, select, spawn, takeEvery } from 'redux-saga/effects';
+import { Api } from '../../../../services/config/Api';
+import {
+  TWINGRAPH_QUERIES_DELAY,
+  TWINGRAPH_QUERY_MAX_RETRIES,
+  TWINGRAPH_QUERY_RETRY_DELAY,
+} from '../../../../services/config/FunctionalConstants';
+import { DATASET_TWINGRAPH_QUERIES_RESULTS_ACTIONS } from '../../../commons/DatasetConstants';
+
+const getOrganizationId = (state) => state.organization.current?.data?.id;
```

Comment:
just out of curiosity: in other sagas, we passe `organizationId` by the hook, is there a special need to pass by` yield select `in this one or it's just a syntax choice? :slightly_smiling_face: